src/lib/components/TestTimer.svelte

```
@@ -112,9 +110,7 @@
 
 			latestAppliedSyncRequestId = requestId;
 			updateTimeLeft(Math.min(timeLeft, data.time_left));
-		} catch {
-			// Keep the local countdown running if timer sync fails.
-		}
+		} catch {}
```

ESLint

[error] Empty block statement.

(no-empty)